net-im/qtox/metadata.xml

```diff
@@ -10,7 +10,7 @@
 		<name>Proxy Maintainers</name>
 	</maintainer>
 	<use>
-		<flag name="gtk">Adds support for GTK and Status Notifier tray backends</flag>
+		<flag name="notifications">Adds support for GTK Status Notifier tray backends</flag>
```

Comment:
Make it 'notification', please. There are two other packages using 'notification' but none using 'notifications', and I think avoiding plural is preferred anyway.